server/game/core/gameSteps/prompts/ResourcePrompt.ts

```diff
@@ -86,7 +87,20 @@ export class ResourcePrompt extends AllPlayerPrompt {
     }
 
     public override menuCommand(player, arg): boolean {
-        Contract.fail(`Unexpected menu command: '${arg}'`);
+        if (arg === 'done') {
```

Comment:
Can we still have the Contract.fail if the command is something else, since that would mean they somehow clicked a button that doesn't exist

Reply:
Yessir!